apps/src/storage/dataBrowser/PreviewModal.jsx

```diff
@@ -37,6 +48,9 @@ export default connect(
   dispatch => ({
     onClose() {
       dispatch(hidePreview());
+    },
```

Comment:
OnClose and OnImport look the same - do we need both?

Reply:
good call, fixed!